python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/logging.j2

```diff
@@ -33,6 +36,12 @@ no logging monitor
 {%     elif logging.monitor is arista.avd.defined %}
 logging monitor {{ logging.monitor }}
 {%     endif %}
+{%     if logging.event.congestion_drops.interval is arista.avd.defined %}
+logging event congestion-drops interval {{ logging.event.congestion_drops.interval }}
+{%     endif %}
+{%     if logging.event.link_status.global is arista.avd.defined(true) %}
+logging event link-status global
```

Comment:
wondering if we would not rather need the `no` version of this command:
```
wan2-site1(config)#show run se logging
logging event congestion-drops interval 43
logging repeat-messages
no logging event link-status global  <<<<<<<<<<<<<<  show run shows the `no` version
wan2-site1(config)#
wan2-site1(config)#
wan2-site1(config)#logging event link-status global  <<<< enabling it and not rendered
wan2-site1(config)#show run se logging
logging event congestion-drops interval 43
logging repeat-messages
wan2-site1(config)#
```

Reply:
This is a good point. I'm looking over an AVD codebase that had eos_cli adding `logging event link-status global`, but as you pointed out this appears to be the default behavior. Which means this isn't really something I need anymore, since AVD is rendering the configuration as desired. But since I'm here I can refactor this to reflect the actual behavior. Is it good enough to change the default value to `true` and have it only render when `false`?

Reply:
With `eos_cli_config_gen` templates, we want to print the command regardless of the default behaviour when set. So it should print when set to true or false, and no default value.